cpp/src/groupby/new_groupby.cu

```diff
@@ -190,3 +192,185 @@ gdf_error gdf_group_by(gdf_column* in_key_columns[],
 
   return gdf_error_code;
 }
+
+
+
+gdf_error gdf_unique_indices(gdf_size_type num_data_cols,
+                     gdf_column** data_cols_in,
+									   gdf_index_type* unique_indices,
+                     gdf_size_type* num_unique_indices, 
+									   gdf_context* ctxt)
+{
+
+  gdf_size_type nrows = data_cols_in[0]->size;
+  // setup for reduce by key  
+  bool have_nulls = false;
```

Comment:
How about a one-liner? 
```suggestion
  bool const have_nulls{ std::any(data_cols_in, data_cols_in + num_data_cols, [](gdf_column * col){ return col->null_count > 0;} };
```

Reply:
Yeah, a one liner is nicer. Assuming you mean std::any_of

Reply:
Yeah, `any_of`, GitHub needs to add a syntax checker to their in-line code modification reviews :)